edunham intro

edunham opened the pull request:
This blog post has the following features:

- [ ] A GitHub Repository with a polished README
- [ ] A GitHub Repository under the github.com/oktadev account
- [x] A title that's approved by Dev Advocacy
- [x] A URL that's approved by Dev Advocacy
- [x] The content has been ran through Grammarly (https://www.grammarly.com/) (still awaiting grammarly setup in okta dashboard)
- [x] Rendered locally and confirmed that no Markdown typos exist

edunham committed:
edunham author info

edunham committed:
edunham intro blog post

havi commented on _source/_posts/2021-10-08-edunham-intro.md:
```diff
@@ -0,0 +1,31 @@
+type: awareness
+---
+
+Hi, I'm edunham, and I'm an infra person. Our buzzwords include DevOps, Cloud Operations, and Site Reliability Engineering, though if you ask any two of us about the difference between those terms, you'll get at least 3 answers. Whatever name it's wearing, doing ops well requires not only the "how" of changing systems, but to the "why" of making the right changes. Although I've always loved helpi ...
```
```suggestion
Hi, I'm edunham, and I'm an infra person. Our buzzwords include DevOps, Cloud Operations, and Site Reliability Engineering, though if you ask any two of us about the difference between those terms, you'll get at least three answers. Whatever name it's wearing, doing ops well requires not only the "how" of changing systems, but also the "why" of making the right changes. Although I've always loved helping spread the information that my colleagues need in order to build the systems that work best for them, I've chickened out of taking the leap into pure advocacy many times, before finally diving into it here at Okta!
```
1) consistency/ spell out numbers under 10 per style guide.  2) missing word? 3) comma for added clarity and a pause.

havi commented on _source/_posts/2021-10-08-edunham-intro.md:
```diff
@@ -0,0 +1,31 @@
+
+As ops for the Rust and Servo teams at [Mozilla](https://research.mozilla.org/), I was again able to play the positive-sum game of sharing the most interesting and useful insights I learned from my unique access to fantastic engineers. But as the sysadmin and speaker roads diverged in that forest of mercurial trees, I stuck with the ops route. 
+
+Missing the conference circuit as growing responsibilities meant less time for talks, I started working on a transition toward advocacy in early 2020. Faced with a choice between making the switch then or building on my ops skills for awhile longer, the engineer in me won out. Switching fields, into a field that was frantically reinventing itself at the time, was just too many simultaneous change ...
```
```suggestion
Missing the conference circuit as growing responsibilities meant less time for talks, I started working on a transition toward advocacy in early 2020. Faced with a choice between making the switch then or building on my ops skills for a while longer, the engineer in me won out. Switching fields, into a field that was frantically reinventing itself at the time, was just too many simultaneous changes to roll out to prod without any staging environment  or CI available.
```
awhile vs a while - i think "for" is the tell, but I checked a couple sources: https://www.grammarly.com/blog/awhile-a-while/

havi commented on _source/_posts/2021-10-08-edunham-intro.md:
```diff
@@ -0,0 +1,31 @@
+
+Missing the conference circuit as growing responsibilities meant less time for talks, I started working on a transition toward advocacy in early 2020. Faced with a choice between making the switch then or building on my ops skills for awhile longer, the engineer in me won out. Switching fields, into a field that was frantically reinventing itself at the time, was just too many simultaneous change ...
+
+Okta's developer advocacy team is in the right place at the right time for me. I get to use [my background](https://edunham.net/pages/talks.html) to help useful information flow out of the company to our community, as well as bringing the community's observations and insights back to contribute to internal decision-making. Our product has made my life better as an engineer better whenever I've wo ...
```
```suggestion
Okta's developer advocacy team is in the right place at the right time for me. I get to use [my background](https://edunham.net/pages/talks.html) to help useful information flow out of the company to our community, as well as bringing the community's observations and insights back to contribute to internal decision-making. Our product has made my life better as an engineer whenever I've worked with it, increasing my knowledge about how to do security well.
```
missiing period. also needed pronoun or change of tense

edunham replied:
oops, missed that on my own proofread! thank you :)

havi commented on _source/_posts/2021-10-08-edunham-intro.md:
```diff
@@ -0,0 +1,31 @@
+
+{% img blog/edunham-intro/travel.jpg alt:"edunham's face" width:"800" %}{: .center-image }
+
+I've been working remotely since 2018, and I put the time that I would otherwise spend commuting into gardening and an assortment of maker projects, which I sometimes mention on [my blog](https://edunham.net/). I'm a volunteer firefighter and emergency medical responder with my local fire department, and I play tuba and sousaphone poorly but enthusiastically on the now-rare occasion that we can g ...
```
```suggestion
I've been working remotely since 2018, and I put the time that I would otherwise spend commuting into gardening and an assortment of maker projects, which I sometimes mention on [my blog](https://edunham.net/). I'm a volunteer firefighter and emergency medical responder with my local fire department, and I play tuba and sousaphone poorly but enthusiastically on the now-rare occasions when we can get enough people together to make some music.
```

havi reviewed (changes requested):
some grammar and usage nits to resolve. thank you.

Commit:
Apply suggestions from code review

thank you for editing well, Havi! it's a delight to catch these bad habits of mine and learn to reduce them :)

Co-authored-by: havi hoffman <havi.hoffman@gmail.com>

havi committed:
Merge branch 'main' into edunham-intro

edunham committed:
add social image and update date

edunham committed:
Merge branch 'edunham-intro' of github.com:edunham/okta-blog into edunham-intro

havi reviewed (approved):
Comments addressed.
